packages/notification-email-sender/src/services/notificationEmailSenderService.ts

```diff
@@ -152,6 +153,28 @@ function retrieveLatestPublishedDescriptor(eservice: EService): Descriptor {
   return latestDescriptor;
 }
 
+/*
+  The following function skip TooManyRequestsException error thrown by AWS SES client
+  for more details: https://aws.amazon.com/blogs/developer/service-error-handling-modular-aws-sdk-js/
+*/
+async function sendEmail(
+  sesEmailManager: EmailManagerSES,
+  mailOptions: Mail.Options,
+  logger: Logger
+): Promise<void> {
+  try {
+    await sesEmailManager.send(mailOptions);
+  } catch (err) {
+    if (err instanceof AllowedSESErrors) {
+      logger.warn(
+        `AWS SES error with name ${err.name} was thrown, it will not considered as fatal, error email notification could be lost; Error details: ${err.message}`
```

Comment:
```suggestion
        `AWS SES error with name ${err.name} was thrown, it will not be considered fatal to avoid losing email notification; Error details: ${err.message}`
```